assets/js/lib/test-utils/factories/advisoryErrata.js

```diff
@@ -2,23 +2,46 @@ import { faker } from '@faker-js/faker';
 import { Factory } from 'fishery';
 import { advisoryType } from './relevantPatches';
 
+const buildFixes = (size) =>
+  Object.fromEntries(
+    new Array(size)
+      .fill(0)
+      .map(() => [
+        faker.number.int({ min: 1, max: 65536 }),
+        faker.lorem.sentence(),
+      ])
+  );
+
+const buildCVE = () =>
+  `CVE-${faker.number.int({ min: 1991, max: 2024 })}-${faker.number.int({
+    min: 0,
+    max: 9999,
+  })}`;
+
```

Comment:
Could we actually have these as factories? So you could just `.buildList`

Reply:
I did change a few things to try and get some of the benefits factories are giving us. Please take a look at the updated version